AWS E2E Jenkins tests failing

```
<snip>
[2022-07-21 18:34:12,344] [[1m[38;5;142mlibcloudforensics.providers.gcp.internal.storagetransfer[0m] INFO     [38;5;15mTransferred 1/1 files (1073741824/1073741824 bytes).[0m
[2022-07-21 18:34:12,344] [[1m[38;5;142mlibcloudforensics.providers.gcp.internal.storagetransfer[0m] INFO     [38;5;15mSkipped 0 files (0 bytes).[0m
[2022-07-21 18:34:13,143] [[1m[38;5;18mGCSToGCEImage       [0m] INFO     [38;5;15mCreating IAM role image_build_role_96601[0m
[2022-07-21 18:34:13,744] [[1m[38;5;18mGCSToGCEImage       [0m] INFO     [38;5;15mApplying permissions for role image_build_role_96601[0m
[2022-07-21 18:34:16,833] [[1m[38;5;18mGCSToGCEImage       [0m] INFO     [38;5;15mPausing 30 seconds to allow permissions to propagate[0m
[2022-07-21 18:34:47,959] [[1m[38;5;62mlibcloudforensics.providers.gcp.internal.common[0m] INFO     [38;5;15mBuild started, logs bucket: gs://472040535346.cloudbuild-logs.googleusercontent.com, logs URL: [https://console.cloud.google.com/cloud-build/builds/db216508-581a-439d-980d-bab0ee1ac86d?project=472040535346[0m](https://console.cloud.google.com/cloud-build/builds/db216508-581a-439d-980d-bab0ee1ac86d?project=472040535346%1B[0m)
[2022-07-21 18:37:12,680] [[1m[38;5;31mlibcloudforensics.providers.gcp.internal.compute[0m] INFO     [38;5;15mImage gs://dftw-aws-gcp-e2e-test-bucket-4n6/snap-0fd5eab1794d93980/image.bin imported as GCE image snap-0fd5eab1794d93980-image-bin.[0m
[2022-07-21 18:37:12,681] [[1m[38;5;18mGCSToGCEImage       [0m] INFO     [38;5;15mDeleting IAM role image_build_role_96601[0m
[2022-07-21 18:37:13,179] [[1m[38;5;32mGCEDiskFromImage    [0m] INFO     [38;5;15m[ThreadPoolExecutor-3_0] Creating disk from image snap-0fd5eab1794d93980-image-bin[0m
[2022-07-21 18:37:25,438] [[1m[38;5;32mGCEDiskFromImage    [0m] INFO     [38;5;15m[ThreadPoolExecutor-3_0] Disk snap-0fd5eab1794d93980-image-bin-nazvg-20220721183713 finished creation[0m
Cleaning up after test...
Deleting AWS EBS Snapshot snap-0fd5eab1794d93980
Deleting AWS S3 object s3://dftimewolf-aws-gcp-e2e-test-bucket/snap-0fd5eab1794d93980/image.bin
Deleting AWS S3 object s3://dftimewolf-aws-gcp-e2e-test-bucket/snap-0fd5eab1794d93980/log.txt
Deleting AWS S3 object s3://dftimewolf-aws-gcp-e2e-test-bucket/snap-0fd5eab1794d93980/hlog.txt
Deleting AWS S3 object s3://dftimewolf-aws-gcp-e2e-test-bucket/snap-0fd5eab1794d93980/mlog.txt
Deleting GCS object gs://dftw-aws-gcp-e2e-test-bucket-4n6/snap-0fd5eab1794d93980/image.bin
Deleting GCE Image snap-0fd5eab1794d93980-image-bin
Deleting GCE Disk snap-0fd5eab1794d93980-image-bin-nazvg-20220721183713
F
======================================================================
FAIL: testRunRecipe (__main__.AWSToGCPForensicsEndToEndTest)
Tests the recipe for AWS->GCP disk copy.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/workspace/dftimewolf-aws-to-gcp-e2e/tests/e2e/aws_disk_forensics.py", line 208, in testRunRecipe
    self.assertIn(d.name, real_gce_disk_names)
AssertionError: 'snap-0fd5eab1794d93980-image-bin-nazvg-20220721183713' not found in ['5084144119340819558', '5848529718311486915', '3583154603890529736', '956054784034698985', '2802575775657841868', '4505882508725015753', '5952339000511649221', '8355851029636408117', '2205560356863958590', '6569913480891030040', '5459011742563786769', '6189441330057534756', '5516913807302210567', '7100588770406391102', '5564947544711324135', '9204403888188437992', '8832992665662661174']
```

Looks like snapshot names changed?

--- a/tests/e2e/aws_disk_forensics.py
+++ b/tests/e2e/aws_disk_forensics.py
@@ -201,8 +201,8 @@ class AWSToGCPForensicsEndToEndTest(unittest.TestCase):
     self.assertEqual(len(self.test_state.GetContainers(containers.AWSVolume)),
         len(self.test_state.GetContainers(containers.GCEDisk)))
 
-    real_gce_disk_names = list(
-        compute.GoogleCloudCompute(self.gcp_project_id).Disks().keys())
+    disks = compute.GoogleCloudCompute(self.gcp_project_id).Disks()
+    real_gce_disk_names = [disks[k].name for k in disks.keys()]
 
     for d in self.test_state.GetContainers(containers.GCEDisk):
       self.assertIn(d.name, real_gce_disk_names)

